ros/src/waypoint_updater/waypnt_updater.py

```diff
@@ -191,13 +191,16 @@ def dyn_vars_cb(self, config, level):
                           .format(old_default_velocity,
                                   config['dyn_default_velocity']))
 
-            if config['dyn_default_velocity'] > self.max_velocity:
+            if config['dyn_default_velocity'] > 0.96 * self.max_velocity:
                 rospy.logwarn("waypoint_updater:dyn_vars_cb default_velocity "
-                              "limited to max_velocity {:3.2f}m/s"
-                              .format(self.max_velocity))
-                self.default_velocity = self.max_velocity * 0.975
+                              "limited to 0.96 * max_velocity = {:3.2f}m/s"
+                              .format(self.max_velocity*0.96))
+                self.default_velocity = self.max_velocity * 0.96
```

Comment:
Should we just set this `self.default_velocity = self.max_velocity * 0.96` to be the case directly in the waypoints callback where self.max_velocity is determined?  Then there is no need for `dyn_default_velocity` to be used.